### DISCUSSION TITLE
How to disable highlighting while editing?

### DISCUSSION THREAD

**Discussion by sovedus** [helix-editor/helix#7675] [Q&A] [answered]
Configuration unchanged.

![Screenshot_2023-07-19_12-37-06](https://github.com/helix-editor/helix/assets/121684127/a9491f43-70b7-49f2-98a7-316ac7580189)

**Comment by pascalkuthe** [comment, marked as answer]
This is not bighkightin but the selection. In append mode helix always extends the selection. This is part of the core editing mode and there is no way to change that (and we won't don't intent to)
